p2p/p2p.proto

```diff
@@ -45,6 +45,23 @@ enum AccessType {
   ACCESS_TYPE_ACCESS_TOKEN_VERIFICATION = 3;
 }
 
+message AccessToken {
```

Comment:
Sorry I'm only adding this now, but given the accesstokens are actually stored in the core document, I think it would be good to move the message there as well.